server/registry/server.go

```diff
@@ -29,12 +32,13 @@ import (
 
 // Config configures the registry server.
 type Config struct {
-	Database  string
-	DBConfig  string
-	LogLevel  string
-	LogFormat string
-	Notify    bool
-	ProjectID string
+	Database         string
+	DBConfig         string
+	LogLevel         string
+	LogFormat        string
+	Notify           bool
+	ProjectID        string
+	APIClientOptions []option.ClientOption
```

Comment:
Is this specifically for pubsub? If so, maybe the name should reflect that? As it is it's kind of inscrutable (e.g. "why would I have an API client in the server?" "are APIClientOptions extra options that clients need to pass to the server?").

Reply:
Not exactly, ClientOptions are set of options for any Google API clients (see: https://pkg.go.dev/google.golang.org/api/option#ClientOption), but yes, PubSub is the only API we use here.